modules/programs/firefox.nix

```diff
@@ -264,6 +270,11 @@ in
           mkIf (profile.settings != {} || profile.extraConfig != "") {
             text = mkUserJs profile.settings profile.extraConfig;
           };
+
+        ".mozilla/firefox/${profile.path}/handlers.json" =
+          mkIf (profile.handlers != {}) {
+            text = builtins.toJSON profile.handlers;
+          };
```

Comment:
Please move this into a separate PR.

Reply:
I will remove this altogether, it is not yet to be published.